deployment/helm/resource-policy/topology_aware_values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for nri-plugins.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+---
+image:
+  name: ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware
+  # tag, if defined will use the given image tag, otherwise Chart.AppVersion will be used
+  tag: unstable
+  pullPolicy: Always
+container:
+  name: nri-resource-policy-topology-aware
```

Comment:
Any reason to have this?  I'd suggest just to drop

```suggestion
```

Reply:
we name containers differently for each plugin. Since we have a single set of templates to be updated by each plugin, we need it here.

Reply:
Why would you need to rename containers differently in different policy deployments. cant you just name it like `policy` or smth?

Reply:
yes, it doesn't make sense indeed. Will remove it.